src/worldcereal/rdm_api/rdm_interaction.py

```diff
@@ -169,12 +211,104 @@ def _get_download_urls(
 
         return urls
 
+    def get_crop_counts(
+        self,
+        ewoc_codes: Optional[List[int]] = None,
+        collection_ids: Optional[List[str]] = None,
```

Comment:
Personally i don't like the duplication overhead of having the option here to give the collections already, or to use this as entry point to still make the query. So I would either not put the collection_ids argument here and always make the request, of put it here as compulsory argument so the actual request already needs to have taken place.

Reply:
good point. 
I was already having the final end user in mind, who would like to write as little python code as possible to get what he wants. 
Having the ability to retrieve samples or retrieve crop counts in a one-liner seemed appealing to me in this respect.
I will simplify and remove the duplication